compiler/circle-mpqsolver/src/CircleMPQSolver.cpp

```diff
@@ -54,6 +59,16 @@ int entry(int argc, char **argv)
     .required(true)
     .help("Input float model with min max initialized");
 
+  arser.add_argument("--input_quantization")
+    .type(arser::DataType::STR)
+    .default_value("uint8")
+    .help("Type of input quantization");
+
+  arser.add_argument("--output_quantization")
+    .type(arser::DataType::STR)
+    .default_value("uint8")
+    .help("Type of output quantization");
```

Comment:
Could you change the argument name?

```suggestion
  arser.add_argument("--input_dtype")
    .type(arser::DataType::STR)
    .default_value("uint8")
    .help("Data type of quantized model's inputs (default: uint8)");

  arser.add_argument("--output_dtype")
    .type(arser::DataType::STR)
    .default_value("uint8")
    .help("Data type of quantized model's outputs (default: uint8)");
```